index.html

```diff
@@ -395,6 +395,23 @@ <h2>Security Considerations</h2>
   </p>
 </section>
 
+
+<section class="informative">
+  <h2>Internationalization Considerations</h2>
+  <p>
+    Technical implementers should take into account several
+    internationalization considerations when publishing data described
+    in this specification. Disregarding internationalization hampers the
+    production and consumption of data across diverse languages and
+    societies, thereby restricting the applicability and significantly
+    reducing the value of the specification as a standard.
+  </p>
+  <p>
+    Implementers are advised to note and abide by all
+    internationalization considerations called out in the
+    [[VC-DATA-MODEL]].
```

Comment:
Are there W3C Internationalization spec(s) that we should also directly reference?